### PR TITLE
add: `setconf`

### PR DESCRIPTION
**Pull request by Xdavius**

### CODE REVIEW

File: packages/setconf/setconf.pacscript [hunk 1]
```diff
@@ -0,0 +1,18 @@
+pkgname="setconf"
+pkgver="0.7.7"
+pkgdesc='Utility for easily changing settings in configuration files'
+arch=("any")
```
Comment (Elsie19) [inline packages/setconf/setconf.pacscript:4]:
```suggestion
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/setconf/setconf.pacscript [hunk 2]
```diff
@@ -0,0 +1,17 @@
+pkgname="setconf"
+pkgver="0.7.7"
+pkgdesc='Utility for easily changing settings in configuration files'
+url='https://setconf.roboticoverlords.org/'
+depends=("python3" "python-is-python3")
+source=("https://setconf.roboticoverlords.org/${pkgname}-${pkgver}.tar.xz"{,.asc})
+validpgpkeys=('962855F072C7A01846405864FCF3C8CB5CF9C8D4')
+sha256sums=('19315574540b3181fec31a4059b9e058381e0192317f153d181e7e7e2aa84d86'
+  'SKIP')
```
Comment (Elsie19) [inline packages/setconf/setconf.pacscript:9]:
```suggestion
sha256sums=('19315574540b3181fec31a4059b9e058381e0192317f153d181e7e7e2aa84d86' 'SKIP')
```

**Review by Elsie19 — CHANGES_REQUESTED** [review]

File: packages/setconf/setconf.pacscript (hunk 2 above)
Comment (Xdavius) [reply packages/setconf/setconf.pacscript:9]: There are 2 sources.

File: packages/setconf/setconf.pacscript [hunk 3]
```diff
@@ -0,0 +1,16 @@
+pkgname="setconf"
+pkgver="0.7.7"
+pkgdesc='Utility for easily changing settings in configuration files'
+url='https://setconf.roboticoverlords.org/'
+depends=("python3" "python-is-python3")
+source=("https://setconf.roboticoverlords.org/${pkgname}-${pkgver}.tar.xz"{,.asc})
+validpgpkeys=('962855F072C7A01846405864FCF3C8CB5CF9C8D4')
```
Comment (oklopfer) [inline packages/setconf/setconf.pacscript:7]: we dont use this variable

File: packages/setconf/setconf.pacscript [hunk 4]
```diff
@@ -0,0 +1,16 @@
+pkgname="setconf"
+pkgver="0.7.7"
+pkgdesc='Utility for easily changing settings in configuration files'
+url='https://setconf.roboticoverlords.org/'
+depends=("python3" "python-is-python3")
+source=("https://setconf.roboticoverlords.org/${pkgname}-${pkgver}.tar.xz"{,.asc})
```
Comment (oklopfer) [inline packages/setconf/setconf.pacscript:6]: is the signature even needed?

**Review by oklopfer — CHANGES_REQUESTED** [review]

**Comment by Xdavius** [comment]
No more .asc
No more key.

**Review by Elsie19 — APPROVED** [review]

**Pull request merged**